src/topology/continuous_function/algebra.lean

```diff
@@ -582,6 +582,36 @@ instance has_scalar' {α : Type*} [topological_space α]
   has_scalar C(α, R) C(α, M) :=
 ⟨λ f g, ⟨λ x, (f x) • (g x), (continuous.smul f.2 g.2)⟩⟩
 
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {R : Type*} [semiring R] [topological_space R]
+  [topological_ring R] : has_continuous_mul C(α, R) :=
+  ⟨begin
+    refine continuous_of_continuous_uncurry _ _,
+    change continuous ((λ (x : R × R), x.1 * x.2) ∘
+      (λ (x : (C(α, R) × C(α, R)) × α), (x.1.1 x.2, x.1.2 x.2))),
+    refine (continuous_fst.comp (continuous.prod_mk _ _)).mul
+      (continuous_snd.comp (continuous.prod_mk _ _)),
+    any_goals { change continuous ((λ (x : C(α, R) × α), x.1 x.2) ∘
+        (λ (x : (C(α, R) × C(α, R)) × α), (x.1.1, x.2))),
+      refine continuous.comp continuous_ev
+        ((continuous_fst.comp continuous_fst).prod_mk continuous_snd), },
+    any_goals { change continuous ((λ (x : C(α, R) × α), x.1 x.2) ∘
+        (λ (x : (C(α, R) × C(α, R)) × α), (x.1.2, x.2))),
+    refine continuous.comp continuous_ev
+      ((continuous_snd.comp continuous_fst).prod_mk continuous_snd), },
+   end⟩
+
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {R : Type*} [semiring R] [topological_space R]
+  [topological_ring R] :
+  has_continuous_smul R C(α, R) :=
```

Comment:
Please move this to the section on modules, and state as `has_continuous_smul R C(α, M)` for `M` with a continuous `R`-action.

Reply:
Done